assignment02/js/script.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function(){
+var femalename1;
+femalename1= "Haley"
```

Comment:
Interesting! You've assigned a default value of sorts to the variables before prompting.